src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -51,7 +51,7 @@ class CargoTestEventsConverter(
         handleStartMessage(text)
 
         val jsonObject = try {
-            val escapedText = text.replace(DOCTEST_PATH_RE) { it.value.replace("\\", "\\\\") }
+            val escapedText = text.replace(NAME_RE) { it.value.replace("\\", "\\\\") }
```

Comment:
Let's create an issue in rust or cargo repo to fix this bug and add link to is here

Reply:
BTW looks like it doesn't work with properly escaped backslashes - it will double them. Can we do nothing with correct escaped backslashes?

Reply:
https://github.com/rust-lang/rust/issues/76151

Reply:
> BTW looks like it doesn't work with properly escaped backslashes - it will double them. Can we do nothing with correct escaped backslashes?

It's not possible to open a file/directory with backslash in name from IDE.